CONTRIBUTING.md

```diff
@@ -210,12 +210,17 @@ $ TRAVIS_OS_NAME=linux DISTRO=ubuntu DISTRO_VERSION=18.04 ./ci/travis/install-li
 ```
 
 Once you create the image for a given combination of `DISTRO` and
-`DISTRO_VERSION`, you can compile the code multiple times, for example:
+`DISTRO_VERSION`, you can reuse it for many builds, with different options.
+For example, this (rather longish)
+command will verify that your code is formatted correctly, run the
+clang-tidy(1) checks, verify the Doxygen documentation can be generated, and run
+both the unit and integration tests:
 
 ```console
-# Also run from google-cloud-cpp:
-$ TRAVIS_OS_NAME=linux DISTRO=ubuntu DISTRO_VERSION=18.04 \
-      CXX=clang++ CC=clang BUILD_TYPE=Debug ./ci/travis/build-linux.sh
+CHECK_STYLE=yes GENERATE_DOCS=yes \
+    CMAKE_FLAGS=-DGOOGLE_CLOUD_CPP_CLANG_TIDY=yes \
+    DISTRO=ubuntu DISTRO_VERSION=18.04 CXX=clang++ CC=clang NCPU=$(nproc) \
+    TRAVIS_OS_NAME=linux ./ci/travis/build-linux.sh
```

Comment:
Any reason `BUILD_TYPE=Debug` is omitted?

Reply:
No.  Would it be good to include it?  Why?

Reply:
Consistency since it's used further up the document but also since in most projects, debug is the default so they may not realize release is used. Your latest change addresses that, however.